src/content/News.md

```diff
@@ -1,3 +1,20 @@
-# News
+# MIND News
+Apps.digitalpsych.org is officially MIND - the Mhealth Index and Navigation Database. The mission of MIND is to make mental health tools more accessible through connecting users with tools that suit their unique needs and preferences. From presentations to partnerships, check out how MIND is working to advance digital mental health. 
 
-There are no new annoncements at this time.
\ No newline at end of file
+# Partnership with BridgingApps 
+BridgingApps is a website of Easter Seals Greater Houston. The Easter Seals network is the largest provider of disability services in North America, outside of the government. Created in 2010, BridgingApps aims to help parents of children with disabilities shortcut the process of finding appropriate apps to help their children learn, progress in therapy, and enhance their lives.  With over 3000 apps now represented on the database, Bridging Apps serves children, adults, caregivers, veterans, among other populations. Check out more about the collaboration here: https://www.bridgingapps.org/2020/10/bridgingapps-collaborates-with-harvard-digital-psychiatry-team-to-highlight-new-tool-for-mental-health-apps/
+
+# FTC Presentation
+The team at the Division of Digital Psychiatry presented at the Federal Trade Commission's Privacy Con in July of 2020 alongside other leading researchers of health privacy and health technology. For more information, as well as a recording of the presentation, check out this link: https://www.ftc.gov/news-events/events-calendar/privacycon-2020 and this link: https://www.natlawreview.com/article/ftc-privacycon-2020-health-apps-area-ftc-focus-going-forward. 
+
+# NAMI's Ask the Expert
+The National Alliance on Mental Illness (NAMI) is the largest grassroots mental health organization in the United States. In May of 2020, Dr. John Torous of the Division of Digital Psychiatry led an "Ask the Expert" session about how to choose a mental health app, introducing the app evaluation model and database to hundreds of webinar attendees. The session can be viewed at this link: https://nami.org/Blogs/NAMI-s-Ask-the-Expert/2020/NAMI-s-Ask-the-Expert-Webinar-What-to-Know-Before-Using-a-Mental-Health-App.
+
+# MIND Featured by STAT News
+The popular health outlet STAT news featured our database in July of 2020: https://www.statnews.com/2020/07/10/new-tool-sort-mental-health-app/. 
```

Comment:
If you do something like this, the user will just see the underlined word instead of the entire link: 

**How you enter into the editor:**
`The popular health outlet STAT news featured our database in July of 2020 <a href="https://www.statnews.com/2020/07/10/new-tool-sort-mental-health-app/" target="_blank">Click here to read.</a>`

**How the user will see it:** 
The popular health outlet STAT news featured our database in July of 2020.  <a href="https://www.statnews.com/2020/07/10/new-tool-sort-mental-health-app/" target="_blank">Click here to read.</a>